src/games/strategy/engine/ClientFileSystemHelper.java

```diff
@@ -135,7 +132,7 @@ public static File getUserRootFolder() {
     final File userHome = new File(System.getProperties().getProperty("user.home"));
     // the default
     File rootDir;
-    if (GameRunner.isMac()) {
+    if (System.getProperties().getProperty("os.name").toLowerCase().contains("mac")) {
```

Comment:
Can we accomplish the same logic without using system properties? Perhaps maybe just check if the mac directory is there first, if so use that, otherwise use the other value?

Do we use GameRunner.isMac() in other places, does it blow up there as well for you too? Could be a bit of a landmine waiting for you to execute it.